%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Daily Laboratory Book
% LaTeX Template
%
% This template has been downloaded from:
% http://www.latextemplates.com
%
% Original author:
% Frank Kuster (http://www.ctan.org/tex-archive/macros/latex/contrib/labbook/)
%
% Important note:
% This template requires the labbook.cls file to be in the same directory as the
% .tex file. The labbook.cls file provides the necessary structure to create the
% lab book.
%
% The \lipsum[#] commands throughout this template generate dummy text
% to fill the template out. These commands should all be removed when 
% writing lab book content.
%
% HOW TO USE THIS TEMPLATE 
% Each day in the lab consists of three main things:
%
% 1. LABDAY: The first thing to put is the \labday{} command with a date in 
% curly brackets, this will make a new page and put the date in big letters 
% at the top.
%
% 2. EXPERIMENT: Next you need to specify what experiment(s) you are 
% working on with an \experiment{} command with the experiment shorthand 
% in the curly brackets. The experiment shorthand is defined in the 
% 'DEFINITION OF EXPERIMENTS' section below, this means you can 
% say \experiment{pcr} and the actual text written to the PDF will be what 
% you set the 'pcr' experiment to be. If the experiment is a one off, you can 
% just write it in the bracket without creating a shorthand. Note: if you don't 
% want to have an experiment, just leave this out and it won't be printed.
%
% 3. CONTENT: Following the experiment is the content, i.e. what progress 
% you made on the experiment that day.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[idxtotoc,hyperref,openany,oneside]{labbook} % 'openany' here removes the gap page between days, erase it to restore this gap; 'oneside' can also be added to remove the shift that odd pages have to the right for easier reading

\usepackage[ 
  backref=page,
  pdfpagelabels=true,
  plainpages=false,
  colorlinks=true,
  bookmarks=true,
  pdfview=FitB]{hyperref} % Required for the hyperlinks within the PDF
  
\usepackage{booktabs} % Required for the top and bottom rules in the table
\usepackage{float} % Required for specifying the exact location of a figure or table
\usepackage{graphicx} % Required for including images
\usepackage{lipsum} % Used for inserting dummy 'Lorem ipsum' text into the template

\usepackage{listings}
\usepackage{color}

\definecolor{green}{rgb}{0,0.5,0}
\definecolor{gray}{rgb}{0.5,0.5,0.5}
\definecolor{mauve}{rgb}{0.58,0,0.82}
\definecolor{red}{rgb}{0.8,0,0}

\lstset{frame=tb,
  language=Python,
  aboveskip=3mm,
  belowskip=3mm,
  showstringspaces=false,
  columns=flexible,
  basicstyle={\small\ttfamily},
  numbers=none,
  numberstyle=\tiny\color{gray},
  keywordstyle=\color{mauve},
  commentstyle=\color{red},
  stringstyle=\color{green},
  breaklines=true,
  breakatwhitespace=true,
  tabsize=3
}

\newcommand{\HRule}{\rule{\linewidth}{0.5mm}} % Command to make the lines in the title page
\setlength\parindent{0pt} % Removes all indentation from paragraphs

%----------------------------------------------------------------------------------------
%	DEFINITION OF EXPERIMENTS
%----------------------------------------------------------------------------------------

\newexperiment{QIIME}{QIIME2 Microbiome}
\newexperiment{Genome}{Genomic Comparisons}
\newexperiment{HMM}{Creating and Using HMMs for Sequence Identification}
\newexperiment{CLSI_Database}{Creation of rpoB, HSP65, and 16S Databases}
\newexperiment{CAMI_Filter}{Viral Contig Identification Study}
\newexperiment{BRASS}{BRASS Asthmatic Microbiome Study}
\newexperiment{gene_g}{Genome Gene Prediction}
\newexperiment{gene_m}{Metagenomic Gene Prediction}
%\newexperiment{shorthand}{Description of the experiment}

%---------------------------------------------------------------------------------------

\begin{document}

%----------------------------------------------------------------------------------------
%	TITLE PAGE
%----------------------------------------------------------------------------------------

\frontmatter % Use Roman numerals for page numbers
\title{
\begin{center}
\HRule \\[0.4cm]
{\Huge \bfseries Laboratory Pipelines \\[0.5cm] \Large} % Degree
\HRule \\[1.5cm]
\end{center}
}
\author{\Huge Cody Glickman \\ \\ \LARGE cody.glickman@ucdenver.edu \\[2cm]} % Your name and email address
\date{2018} % Beginning date
\maketitle

\tableofcontents

\mainmatter % Use Arabic numerals for page numbers

%----------------------------------------------------------------------------------------
%	LAB BOOK CONTENTS
%----------------------------------------------------------------------------------------

\labday{BLAST Usage}

\textbf{Make BLAST Database}

-dbtype [prot/nucl] protein or nuclotide, 
use \begin{verbatim} -parse_seqids \end{verbatim} to enable retrieval 
of sequences by sequence identifiers. 


\begin{verbatim}
makeblastdb -in [input_seq] -dbtype prot -out Combined_VF -title "Combined_VF"

blastp -db [Path_to_Database] -out results.txt -outfmt 6 -query [input_seqs]
-max_target_seqs 2 -perc_identity 95 
\end{verbatim}

\textbf{Output format}
Useful \href{https://www.biostars.org/p/88944/}{outfmt link} on biostars. Formats of interest include 5 XML Blast output, 6 tabular output, 10 comma-seperated values

Standard columns in outfmt 6:
\begin{verbatim}
 1.	 qseqid	 query (e.g., gene) sequence id
 2.	 sseqid	 subject (e.g., reference genome) sequence id
 3.	 pident	 percentage of identical matches
 4.	 length	 alignment length
 5.	 mismatch	 number of mismatches
 6.	 gapopen	 number of gap openings
 7.	 qstart	 start of alignment in query
 8.	 qend	 end of alignment in query
 9.	 sstart	 start of alignment in subject
 10.	 send	 end of alignment in subject
 11.	 evalue	 expect value
 12.	 bitscore	 bit score
 
 Additional columns can be specific by -outfmt "6 std qlen"
\end{verbatim}

\textbf{Custom Output Format} \\
-outfmt "6 qseqid sseqid pident length mismatch evalue bitscore stitle"

\textbf{Columns in Tab-seperated format} \\
qseqid  sseqid  pident  length  mismatch  gapopen qstart  qend  sstart  send  evalue  bitscore

\labday{Microbiome Analysis}
\experiment{QIIME}

Cite vHMM \href{http://portal.nersc.gov/dna/microbial/prokpubs/EarthVirome_DP/}{Earth Virome Project}. 

\begin{verbatim}
## Load required libraries
module load qiime2
source activate qiime2-2017.12

## Create demux sequences
qiime tools import --type 'SampleData[PairedEndSequencesWithQuality]' 
--input-path manifest_file.csv --output-path [paired-end-demux] 
--source-format PairedEndFastqManifestPhred33

## Perform Dada2 Processing 
bsub qiime dada2 denoise-paired --i-demultiplexed-seqs [paired-end-demux]
--p-trunc-len-f 280 --p-trunc-len-r 279 --p-max-ee Inf --p-chimera-method 
pooled --p-n-threads 0 --o-representative-sequences [rep-seqs] 
--o-table [table]

## Summarize Table
bsub qiime feature-table summarize --i-table table.qza --o-visualization table.qzv

## Assign Taxonomy
bsub qiime feature-classifier classify-sklearn --i-classifier [Database File]
--i-reads [rep-seqs] --o-classification [taxonomy]

## Visualize Taxonomy
qiime metadata tabulate --m-input-file ./Qiime_demo/taxonomy.qza --o-visualization [taxonomy.qzv]


\end{verbatim}





\labday{Gene Prediction Methodolgy}

Utilize \href{https://github.com/hyattpd/Prodigal}{Prodigal} to predict genes in prokaryotes.

\experiment{gene_g}

\begin{verbatim}
prodigal -i [input_seqs] -o my_genes -a my_proteins.faa
\end{verbatim} 

\experiment{gene_m}

\begin{verbatim}
prodigal -i [input_seqs] -o my_genes -a my_proteins.faa -p meta
\end{verbatim} 

\begin{lstlisting}
## Hello World 
print(x)
\end{lstlisting} 


\labday{BASHING}

\textbf{PHASTER Summary to Table Parser}
Input File is summary.txt from Phaster and the output table is output.txt

The following workflow is wrapped into table\_parsing\_workflow.sh the folder with the summary file shoud be in the same directory as the workflow and python script.

\begin{verbatim}
sed -i '' -n '/REGION/,$p' $table

sed -i '' -e 's/-//g' $table

sed -i '' -e 's/REGION_POSITION/ASSEMBLY_CONTIG_NUMBER
CONTIG_LENGTH      CONTIG_DEPTH/g' $table

awk -v OFS="\t" '$1=$1' $table > output.txt

./Table_Parsing.py output.txt $table
\end{verbatim} 



\end{document}