%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Daily Laboratory Book
% LaTeX Template
%
% This template has been downloaded from:
% http://www.latextemplates.com
%
% Original author:
% Frank Kuster (http://www.ctan.org/tex-archive/macros/latex/contrib/labbook/)
%
% Important note:
% This template requires the labbook.cls file to be in the same directory as the
% .tex file. The labbook.cls file provides the necessary structure to create the
% lab book.
%
% The \lipsum[#] commands throughout this template generate dummy text
% to fill the template out. These commands should all be removed when 
% writing lab book content.
%
% HOW TO USE THIS TEMPLATE 
% Each day in the lab consists of three main things:
%
% 1. LABDAY: The first thing to put is the \labday{} command with a date in 
% curly brackets, this will make a new page and put the date in big letters 
% at the top.
%
% 2. EXPERIMENT: Next you need to specify what experiment(s) you are 
% working on with an \experiment{} command with the experiment shorthand 
% in the curly brackets. The experiment shorthand is defined in the 
% 'DEFINITION OF EXPERIMENTS' section below, this means you can 
% say \experiment{pcr} and the actual text written to the PDF will be what 
% you set the 'pcr' experiment to be. If the experiment is a one off, you can 
% just write it in the bracket without creating a shorthand. Note: if you don't 
% want to have an experiment, just leave this out and it won't be printed.
%
% 3. CONTENT: Following the experiment is the content, i.e. what progress 
% you made on the experiment that day.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[idxtotoc,hyperref,openany]{labbook} % 'openany' here removes the gap page between days, erase it to restore this gap; 'oneside' can also be added to remove the shift that odd pages have to the right for easier reading

\usepackage[ 
  backref=page,
  pdfpagelabels=true,
  plainpages=false,
  colorlinks=true,
  bookmarks=true,
  pdfview=FitB]{hyperref} % Required for the hyperlinks within the PDF
  
\usepackage{booktabs} % Required for the top and bottom rules in the table
\usepackage{float} % Required for specifying the exact location of a figure or table
\usepackage{graphicx} % Required for including images
\usepackage{lipsum} % Used for inserting dummy 'Lorem ipsum' text into the template

\usepackage{listings}
\usepackage{color}

\definecolor{green}{rgb}{0,0.5,0}
\definecolor{gray}{rgb}{0.5,0.5,0.5}
\definecolor{mauve}{rgb}{0.58,0,0.82}
\definecolor{red}{rgb}{0.8,0,0}

\lstset{frame=tb,
  language=Python,
  aboveskip=3mm,
  belowskip=3mm,
  showstringspaces=false,
  columns=flexible,
  basicstyle={\small\ttfamily},
  numbers=none,
  numberstyle=\tiny\color{gray},
  keywordstyle=\color{mauve},
  commentstyle=\color{red},
  stringstyle=\color{green},
  breaklines=true,
  breakatwhitespace=true,
  tabsize=3
}

\newcommand{\HRule}{\rule{\linewidth}{0.5mm}} % Command to make the lines in the title page
\setlength\parindent{0pt} % Removes all indentation from paragraphs

%----------------------------------------------------------------------------------------
%	DEFINITION OF EXPERIMENTS
%----------------------------------------------------------------------------------------

\newexperiment{VF_HMM}{Identifying Virulence Factors in Phages}
\newexperiment{GRAB}{Genomic Retrieval and BLAST Database Creation}
\newexperiment{Active_Prophage}{Classifying predicted prophages as active or degraded}
\newexperiment{CLSI_Database}{Creation of rpoB, HSP65, and 16S Databases}
%\newexperiment{shorthand}{Description of the experiment}

%---------------------------------------------------------------------------------------

\begin{document}

%----------------------------------------------------------------------------------------
%	TITLE PAGE
%----------------------------------------------------------------------------------------

\frontmatter % Use Roman numerals for page numbers
\title{
\begin{center}
\HRule \\[0.4cm]
{\Huge \bfseries Laboratory Journal \\[0.5cm] \Large} % Degree
\HRule \\[1.5cm]
\end{center}
}
\author{\Huge Cody Glickman \\ \\ \LARGE cody.glickman@ucdenver.edu \\[2cm]} % Your name and email address
\date{Beginning 27 February 2018} % Beginning date
\maketitle

\tableofcontents

\mainmatter % Use Arabic numerals for page numbers

%----------------------------------------------------------------------------------------
%	LAB BOOK CONTENTS
%----------------------------------------------------------------------------------------


\labday{Week of 26 February 2018}

\experiment{VF_HMM}

Downloaded phage protein data searching by taxonomy from Uniprot/TrEmbL. Dowloaded three datasets Claudioviruses, Ligamenvirales, and Unclassified. Downloaded viral contigs from vHMM \href{http://portal.nersc.gov/dna/microbial/prokpubs/EarthVirome_DP/}{Earth Virome Project}. 

Metagenomic gene prediction through prodigal downloaded from conda "conda install prodigal"

\begin{verbatim}
prodigal -i mVGs_sequences_v2.fna -o my_genes -a my_proteins.faa -p meta
\end{verbatim}

The myproteins.faa file contains the translated predicted genes. This set is applied to the VF HMM and similar to the other datasets, returned no hits with hmmsearch. 

\vspace{0.3cm}

Creating virulence factor blast database to blast against viral contigs. Choosing a E value threshold:


\textbf{Goals}
Finish written GRAB and Abstract for NLM
Done (3/2/18)


\begin{verbatim}
makeblastdb -in Combined_VF.faa -dbtype prot -out Combined_VF -title "Combined_VF"

blastp -db /Users/stronglab2/blastdb/Combined_VF/Combined_VF -out results.txt
-outfmt 6 -query phage_proteins.faa
\end{verbatim}


\begin{lstlisting}
## Hello World 
print(x)
\end{lstlisting} 


%----------------------------------------------------------------------------------------



\labday{Week of 5 March 2018}

\textbf{Goals}
\begin{itemize}
\item Prophage Annotation and VF/ARG Pipeline (Wednesday)
\item ML Pipeline Active Prophage
\item CAMI Data With Conda for Reproducibility
\end{itemize}

\experiment{VF_HMM}
Establishing the BASH pipeline

\begin{verbatim}
1. Prophage Prediction
Input: Contigs.fasta
Output: Prophage Zip Folder

2. Gene Prediction
Input: Sequences of Identified Prophages
Output: Protein Fasta

3. Virulence Factor Identification
Input: Protein Fasta
Output: Proteins called Virulence Factors

\end{verbatim}


\experiment{Lysogenic Pan Genome}

Downloaded phage table from \href{http://phagesdb.org/data/}{PhageDB}. Parse temperate phages from Graham Hatful's List and those that infect Mycobacterium. Save Genbank ID numbers as Numbers.txt in script PhageDBProcessing.Rmd.



Calling GBK Files from nuccore. 

\begin{lstlisting}
## Load GenBankIds and Remove Whitespace
with open("Numbers.txt") as f:
    content = f.readlines()
content = [x.strip() for x in content]

## Call Entrez for Genbank_Ids
for i in range(0,len(content)):
    handle = Entrez.efetch(db="nuccore", id=content[i], rettype="gb")
    filename = 'genbank_files/'+ content[i] + '.gbk'
    out_handle = open(filename, "w")
    out_handle.write(handle.read())
    out_handle.close()
    handle.close()
    print("Saved " + filename)
\end{lstlisting} 

\textbf{Run Core Genome Analysis}

\vspace{0.3cm}

Convert to GFF3

\begin{verbatim}
bp_genbank2gff3.pl --dir pathtofiles
\end{verbatim}


Run Roary

\begin{verbatim}
roary -e --maft -p 8 *.gff
\end{verbatim}


%----------------------------------------------------------------------------------------

\labday{Week of 12 March 2018}


\experiment{GRAB}

Submitted GRAB document to bioRxiv: received resubmission request

Resupply manuscript to MS ID 246553

Include data about functionality compared to other tools or show with an example dataset


%---------------------------------------------------------------------------------------

\labday{Week of 19 March 2018}

\experiment{Active_Prophage}

Met Monday with Graham Hatfull and members of his lab. Confirmed genes to predict lysogenic life cycle. Holins are hard to predict as they are transmembrane proteins and may have small amount of conservation. Holins may be better predicted by k-mer protein groups. 

\begin{itemize}
\item Integrases \\
Involved in phage insertion into host genome
\item ParA - ParB - ParS \\
Involved in extra chromosomal arrangement and replication
\item Repressors \\
Inhibits replication until stimulus 
\end{itemize}

Additional Take-Aways: Portal genes may be targets of bacterial resistance to phages and Abscessus excised phages can be engineered to become lytic.  Excised phages resemble cluster P or N, Abscessus infected by cluster A3\\

Review Kclust and MMseq2 for clustering sequences \\

Check for non synonomous mutations in presence of clustered gene \\

Installed MMSeqs2 for sequence clustering

Downloaded viral HMMs from \href{http://eggnogdb.embl.de/#/app/viruses}{EggNog 4.5}  |  \href{http://vogdb.org/download}{VOGDB}  | \href{http://dmk-brain.ecn.uiowa.edu/pVOGs/downloads.html}{pVOG} \\

\textbf{Process to Filter Viral Lysogeny HMMs from Pfam}

Split the complete Pfam database and rename each file to the Pfam ID 

\begin{verbatim}
python -c "import sys
for i, c in enumerate(sys.stdin.read().split('//')):
    open('out' + str(i), 'w').write(c)" < Pfam.hmm
\end{verbatim}

Removed empty first space from Split:
\begin{verbatim}
for f in *; do grep . $f > $f.hmm; done;
\end{verbatim}

Rename HMMs by Families Identifiers (3rd line of HMMs):
\begin{verbatim}
for f in *.hmm; 
do 
	output=$(awk 'NR == 3 {print $2}' "$f" | cut -f1 -d '.')
	mv "$f" "$output".hmm
done
\end{verbatim}

Filter HMMs by those present in list:
\begin{verbatim}
while read file
do
	mv -v -i "$file".hmm /matched/
done < File_with_names.txt
\end{verbatim}

Format filtered hmms for search against combined lysogenic profiles \\
\textit{Note: Needed to add ending '//' to hmm files to hmmpress effectively}

Perform HMM Search and Create Table Separated
\begin{verbatim}
hmmsearch --tblout [table].txt [model].hmm [sequences].fasta > /dev/null

## Convert tblout to table
chmod +x convert_hmm_tblout_to_tab_seperated_table.sh

./convert_hmm_tblout_to_tab_seperated_table.sh -t [table].txt 
\end{verbatim}


\experiment{CLSI_Database}

\textbf{Query:}
\begin{verbatim}
(("Mycobacterium"[Organism] OR ("Mycobacterium"[Organism] 
OR Mycobacterium[All Fields])) AND rpoB[Title]) AND 
(bacteria[filter] AND biomol_genomic[PROP] AND 
ddbj_embl_genbank[filter] AND ("500"[SLEN] : "5000"[SLEN]))
\end{verbatim}

Muscle Sequence aligner downloaded via bioconda

usage:
More usage examples at \href{http://www.drive5.com/muscle/muscle.html}{Drive5}


\begin{verbatim}
muscle -msf -in seqs.fa -out seqs.msf
\end{verbatim}


%----------------------------------------------------------------------------------------

\labday{Week of 26 March 2018}

\experiment{Active_Prophage}

Thought: determine if focued kmer profile for mycobacteriophage would be ammendable.

\begin{verbatim}
for f in Pfam_Viral_HMMs/*.hmm; do filename=$(echo "${f\%\%.*}"); hmmsearch --tblout $filename.txt $f combined_lysogenic_phages_proteins.fasta > /dev/null; done;
\end{verbatim}

Move output to new folder titled SearchOutput and move into active directory

\begin{verbatim}
for f in Search_Output/*.txt; do ./convert_hmm_tblout_to_tab_seperated_table.sh -t $f; done;
\end{verbatim}




%----------------------------------------------------------------------------------------

\labday{Week of 9 April 2018}

\experiment{GRAB}

Update Database Process

\begin{itemize}

\item Download NCBI Linages from \href{https://gitlab.com/zyxue/ncbitax2lin-lineages/blob/master/lineages-2018-03-12.csv.gz}{Here} as of March 12th. 
\item Download Assembly Summary 

\begin{verbatim}
wget ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/bacteria/assembly_summary.txt
\end{verbatim}

\item Filter Taxa Linage by Kingdom 

The script Filtertaxlinage.R filters both the NCBI taxa lineages and the assembly summary files to include relevant information. \textbf{Note:} included no.rank2 as column in taxa lineages to include strain level identity. 

\end{itemize}

Updating GRAB to include filtering by Strain Level




%----------------------------------------------------------------------------------------

\labday{Week of 16 April 2018}

\experiment{VF_HMM}

Blasted VF Database against Self to identify redundant records between VFDB and Patric VF


\begin{verbatim}
blastp -db /Users/stronglab2/blastdb/Combined_VF/Combined_VF -out results.txt 
 -outfmt 6 -query all_viral_protein.faa 
\end{verbatim}


%----------------------------------------------------------------------------------------
%	FORMULAE AND MEDIA RECIPES
%----------------------------------------------------------------------------------------

\labday{} % We don't want a date here so we make the labday blank

\begin{center}
\HRule \\[0.4cm]
{\huge \textbf{Formulae and Media Recipes}}\\[0.4cm] % Heading
\HRule \\[1.5cm]
\end{center}

%----------------------------------------------------------------------------------------
%	MEDIA RECIPES
%----------------------------------------------------------------------------------------

\newpage

\huge \textbf{Media} \\ \\

\normalsize \textbf{Media 1}\\
\begin{table}[H]
\begin{tabular}{l l l}
\toprule
\textbf{Compound} & \textbf{1L} & \textbf{0.5L}\\
\toprule
Compound 1 & 10g & 5g\\
Compound 2 & 20g & 10g\\
\bottomrule
\end{tabular}
\caption{Ingredients in Media 1.}
\label{tab:med1}
\end{table}

%-----------------------------------------

%\textbf{Media 2}\\ \\

%Description

%----------------------------------------------------------------------------------------
%	FORMULAE
%----------------------------------------------------------------------------------------

\newpage

\huge \textbf{Formulae} \\ \\

\normalsize \textbf{Formula 1 - Pythagorean theorem}\\ \\
$a^2 + b^2 = c^2$\\ \\

%-----------------------------------------

%\textbf{Formula X - Description}\\ \\

%Formula

%----------------------------------------------------------------------------------------

\end{document}